src/content/development/contribution-guide/_index.en.md

```diff
@@ -58,7 +58,35 @@ To be expanded
 
 #### Pre-Planning
 
-To be expanded
+The week before the release is dedicated to planning the next release.
+Planning accounts for [tasks](#tasks) and [bugs](#bugs) which we intend to solve in the next version.
+Planning meetings will be held, focusing on the [Backlog board].
+
+For a [task](#tasks) to be eligible for the next version, it needs to fulfill these requirements:
+
+* Be part of the [Backlog board].
+* Have a `next-version-candidate` label.
+* Have a description detailing **what** is the issue and optionally **how** is it going to be solved.
+* Have an apropriate sizing label, according to amount of work expected for a single person to completely deliver the task:
+  * *Small*: Work is contained in one sprint and is expected to take less than half the sprint.
+  * *Medium*: Work is contained in one sprint and is expected to take most of the sprint.
+  * *Large*: Work is contained within a release (two-three sprints).
+  * *Extra-Large* = Work can't be contained within a release and would span multiple releases.
```

Comment:
```suggestion
  * *Extra-Large* = Work can't be contained within a release and would span multiple releases (or require multiple contributors, if it can be split up appropriately).
```

(this might seem obvious but I think it should be spelled out).

Reply:
I don't mind adding the explanation but generally sizing is for 1 person (obviously we can have more than one person working on it).
Perhaps it's better to add it on L70?

Reply:
@skitt ?

Reply:
OK, the introduction to the list of bullet points should be sufficient.